openbb_terminal/stocks/dark_pool_shorts/dps_controller.py

```diff
@@ -134,6 +136,35 @@ def call_shorted(self, other_args: List[str]):
                 export=ns_parser.export,
             )
 
+    @log_start_end(log=logger)
```

Comment:
@jmaslek means around line 100 where you have `def print_help(self):` add this new `ctb cost to borrow` option 😄